src/test/java/plugins/AnalysisCollectorPluginTest.java

```diff
@@ -329,6 +330,36 @@ public void should_aggregate_warnings_in_dashboard_portlet() {
         assertThat(dashboard, not(hasWarningsFor(job, TASKS, TASKS_ALL)));
     }
 
+    @Test
+    @WithPlugins({"git@2.4.0", "workflow-aggregator@1.10"})
+    public void should_compute_annotations_on_workflow() {
+        MavenInstallation.installMaven(jenkins, "M3", "3.3.3");
+        WorkflowJob job = jenkins.jobs.create(WorkflowJob.class);
+        // TODO: Job.copyResource(String) does not work for WorkflowJob, since it does not owns
+        // a workspace but the build does (actually it could own 0..N workspaces).
```

Comment:
:bug: so override these methods in `WorkflowJob` to do just that. (Or better, implement [my suggestion](https://github.com/jenkinsci/acceptance-test-harness/blob/935342b6b6da2bfe2bc257e5fbccbfc8a03771a5/src/main/java/org/jenkinsci/test/acceptance/po/Job.java#L50).)